README.md

```diff
@@ -18,7 +18,7 @@ mysql-async is no longer supported.
 
 Used for some UI elements (i.e. notifications, progress circle, input), and cache.
 
-### [qtarget](https://github.com/overextended/qtarget)
+### [qtarget](https://github.com/overextended/qtarget) or [ox_target](https://github.com/overextended/ox_target) or [qb_target](https://github.com/qbcore-framework/qb-target)
```

Comment:
```suggestion
### [qtarget](https://github.com/overextended/qtarget) or [ox_target](https://github.com/overextended/ox_target) or [qb-target](https://github.com/qbcore-framework/qb-target)
```
qshit uses - instead of _ for their resource naming